admin_manual/configuration_server/occ_command.rst

```diff
@@ -661,8 +661,11 @@ File operations
  files
   files:cleanup              cleanup filecache
   files:scan                 rescan filesystem
-  files:transfer-ownership   All files and folders are moved to another 
-                             user - shares are moved as well.
+  files:transfer-ownership   All files' and folders' ownerships are moved to another 
+                             user. Outgoing shares are moved as well.
+                             Incoming shares are not moved because the sharing user 
+                             holds the ownership of the respective files and therefore 
+                             owns the decision which actual persons to be shared with.
```

Comment:
Maybe replace **persons** with **users** to be consistent? Not a big deal :)

Reply:
yah, fair enough. Persons is more logical, going down to the intention that swings behind the reason of this behaviour, but users is actually more common.
Fixed it.